nanoFramework.CoreLibrary/System/SpanByte.cs

```diff
@@ -0,0 +1,172 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+//
+
+namespace System
+{
+    /// <summary>
+    /// Provides a type- and memory-safe representation of a contiguous region of arbitrary byte array
+    /// </summary>
+    public struct SpanByte
```

Comment:
It should be OK to have this as `public readonly ref struct SpanByte` after setting the language to 7.3.

Reply:
great :-) adding it!